qemu/qemu-guest-agent/vars.yaml

```
@@ -0,0 +1 @@
+VERSION: "{{ .QEMU_VERSION }}-{{ .BUILD_ARG_TAG }}"
```

```suggestion
VERSION: "{{ .QEMU_VERSION }}"
```

since this doesn't depend on talos kernel version

---

Technically it does depend on the Talos version since a future version will hopefully provide `/sbin/shutdown` so the patch here can be removed, which would then be a breaking change for older Talos versions.

---

Will get the `/sbin/shutdown` for 1.5, so should be good

---

Great to hear, thanks.